Initial preparation for 2.4.0 (#145)
bottlerocket-os/bottlerocket-kernel-kit: CHANGELOG.md

# v2.4.0 (TBD)

## OS Changes

>  * Improve NVIDIA kmods' dependency resolution ([#133])

Reviewer: nit: this reads slightly better IMO
```suggestion
 * Improve dependency resolution for NVIDIA kmods ([#133])
```

>  * Prevent mismatches between the NVIDIA kmod and kernel versions ([#133])

Reviewer: Is this redundant with the line item above?
```suggestion
 * Prevent version mismatches between NVIDIA kmods and kernels ([#133])
```

>  * Strip Open GPU and GRID kernel modules ([#139])

Reviewer: ```suggestion
 * Strip NVIDIA open GPU and GRID kernel modules ([#139])
```

 * Provide nvoptix.bin through NVIDIA kmod 570 for 6.12 kernel ([#141])

>  * Enable cpusets for cgroupsv1 in the 6.12 kernel ([#143])

Reviewer: ```suggestion
 * Enable cpusets for cgroup v1 in the 6.12 kernel ([#143])
```

>  * Prefer LZ4 compression over LZO for ZRAM in 6.123 kernel ([#143])

Reviewer: ```suggestion
 * Prefer LZ4 compression over LZO for zram in the 6.12 kernel ([#143])
```

>  * Provide ext4 support as a loadable module ([#143])

Reviewer: ```suggestion
 * Make ext4 support a module for the 6.12 kernel ([#143])
```
